sidebars.js

```diff
@@ -42,6 +42,7 @@ module.exports = {
         "community/welcome-to-the-community",
         "community/100-days-of-oss",
         "community/faqs",
+        "Community/glossary",
```

Comment:
@Lymah123 the data here is case-sensitive. So this has to be written in lower-case.
It would be very helpful if you run the docs locally to test and see if things work properly in the future. :)

```suggestion
        "community/glossary",
```